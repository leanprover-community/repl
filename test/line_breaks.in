{"cmd": "theorem foo : 1 = 1 := by\n
sorry", "sorries": "individual"}

{"cmd": "theorem bar : 1 = 1 := by\n
/- Some long comment here -/\n
  rfl"}
