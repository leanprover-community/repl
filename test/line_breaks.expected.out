{"sorries":
 [{"proofState": 0,
   "pos": {"line": 2, "column": 0},
   "parentDecl": "foo",
   "goals": ["‚ä¢ 1 = 1"],
   "endPos": {"line": 2, "column": 5}}],
 "messages":
 [{"severity": "warning",
   "pos": {"line": 1, "column": 8},
   "endPos": {"line": 1, "column": 11},
   "data": "declaration uses 'sorry'"}],
 "env": 0}

{"env": 1}

